### PR TITLE
COMMERCE-11957 - When including a dot as part of a Specification Key, the Specification Facet does not work

### PR DESCRIPTION
**Pull request by victorcdp**
[COMMERCE-11957](https://liferay.atlassian.net/browse/COMMERCE-11957)

@andrea-ale-sbarra Can you review this one? Tied to an LPP. Thank you.

### CODE REVIEW

**Comment by victorcdp** [comment]
ci:test:relevant

**Comment by victorcdp** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-product-options-web/src/main/resources/META-INF/resources/edit_cp_specification_option.jsp [hunk 1]
```diff
@@ -53,4 +53,8 @@ else {
 			id="<%= CPSpecificationOptionFormNavigatorConstants.FORM_NAVIGATOR_ID_COMMERCE_PRODUCT_SPECIFICATION_OPTION %>"
 		/>
 	</div>
-</aui:form>
\ No newline at end of file
+</aui:form>
+
+<liferay-frontend:component
+	module="js/edit_cp_option_and_value"
+/>
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-options-web/src/main/resources/META-INF/resources/edit_cp_specification_option.jsp:60]: If you really need this, please make a copy paste of the file and name it **edit_cp_specification_option.js**

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM. 
Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
QA passed. Add automation and resent on https://github.com/liferay-commerce/liferay-portal/pull/4047

**Comment by jaydawu** [comment]
ci:close

**Pull request closed without merging**